templates/agenda/broadcast/index.html.twig

```diff
@@ -5,6 +5,11 @@
 {% block title %}{% trans %}Agenda broadcasts{% endtrans %}{% endblock %}
 
 {% block header %}
+    {% for message in app.flashes('error') %}
+        <div class="alert alert-danger">
+            {{ message }}
+        </div>
+    {% endfor %}
```

Comment:
We should move this to our base template and handle all types of flash messages (cf. https://symfony.com/doc/current/controller.html#flash-messages).

Reply:
Moved to `templates/layout-with-agenda-submenu.html.twig`.